frontend/src/components/Navigator/components/Form.js

```diff
@@ -9,7 +9,7 @@ import { Form as UswdsForm, Button } from '@trussworks/react-uswds';
 import { useForm } from 'react-hook-form';
 
 function Form({
-  initialData, onSubmit, onDirty, saveForm, renderForm,
```

Comment:
Love using `onContinue` here, very clear that we're not submitting the whole process.